aspnet-core/DataManager/Overview.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Overview | DataManager  | ASP.NET Core | Syncfusion
-description: overview 
+description: Learn here about overview in Syncfusion Essential ASP.NET Core DataManager Control, its elements, and more. 
 platform: aspnet-core
 control: DataManager
 documentation: ug
 ---
 
-# DataManager
+# DataManager in ASP.NET Core
```

Comment:
Change like this - Overview of ASP.NET Core DataManager

Reply:
Done